build/images/tf_operator/Dockerfile

```diff
@@ -1,10 +1,13 @@
 FROM registry.access.redhat.com/ubi8-dev-preview/ubi:latest
```

Comment:
We are from the same base image, does the control loop work?

Reply:
This image is multi-arch one. I had tried it on power.

Reply:
@gaocegege  Thanks for your review! :)